MisspelledWord.js

```diff
@@ -0,0 +1,13 @@
+function changeWord (wordMisspelled, correction){

```

Comment:
I'm not familiar with javascript so I'm not sure exactly what's going on here but it looks like code to make red lines appear under misspelled words. This isn't a feature that we talked about implementing but I'll merge it for now in case we decide to implement this at a later time.